source/GameInterface/Serialization/Impl/TextObjectBinaryPackage.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Common.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+using TaleWorlds.Localization;
+
+namespace GameInterface.Serialization.Impl
+{
+    [Serializable]
+    public class TextObjectBinaryPackage : BinaryPackageBase<TextObject>
+    {
+        public TextObjectBinaryPackage(TextObject obj, BinaryPackageFactory binaryPackageFactory) : base(obj, binaryPackageFactory)
+        {
+        }
+
+        public override void Pack()
+        {
+            foreach (FieldInfo field in ObjectType.GetAllInstanceFields())
```

Comment:
Pass an exclude list to `GetAllInstanceFields` for any field with the `CachedData` attribute